src/scopes/activityReport/group.ts

```diff
@@ -2,6 +2,16 @@ import { Op, WhereOptions } from 'sequelize';
 import { sequelize } from '../../models';
 import { idClause } from '../utils';
 
+const constructLiteral = (query: string[], userId: number): string => {
+  const where = idClause(query);
+  return sequelize.literal(`(
+    SELECT "activityReportId" FROM "ActivityRecipients" WHERE "grantId" IN (
+      SELECT "grantId" FROM "GroupGrants" WHERE "groupId" IN (
+        SELECT "id" FROM "Groups" WHERE "id" IN (${where}) AND "userId" = ${userId} OR "isPublic" = true)
+    )
```

Comment:
Because we are skipping the models, we are also not dealing with any validations. I am assuming that's not going to lead to innacurate results, e.g. activity reports that are deleted, but still present in the "ActivityRecipients".

Reply:
The scopes aren't applied without the context of additional logic. As is true for all the existing scopes, we here rely on the individual handlers that they are passed into to apply additional filtering, for example, approved reports only.